test/suite/wttest.py

```diff
@@ -749,6 +760,15 @@ def ttyVerbose(level, message):
     def shortid(self):
         return self.id().replace("__main__.","")
 
+    def sanitized_shortid(self):
+        """
+        Return a name that is suitable for creating file system names.
+        In particular, names with scenarios look like
+        'test_file.test_file.test_funcname(scen1.scen2.scen3)'.
+        So transform '(', but remove final ')'.
+        """
+        return self.shortid().translate(str.maketrans('($[]/','_____', ')'))
```

Comment:
In other parts of my life, name with spaces often cause problems. Maybe replace that with an underscore, too?

Reply:
Sure, good idea.